config/locales/debt_management/debt_management.en.yml

```diff
@@ -1,4 +1,12 @@
 en:
+  meta_title: What to do if your debt management plan has stopped
```

Comment:
i'm surprised this contents of this file is not namespaced eg...
```
en:
  campaigns:
    debt_management:
      translations: go here
```
as all the locale files will get munged together into one hash which I18n lib will use

I think we're lucky this isn't causing bugs or is causing bugs that we are not aware off...

although maybe outside the scope of this ticket this should be noted and fixed